combat-trainer.lic

```diff
@@ -1314,6 +1314,11 @@ class AttackProcess
     pause
     waitrt?
 
+    if reget(20, 'increase the accuracy')
```

Comment:
I generally try to avoid using reget because it's not reliable. Can we instead reset flags if we've finished the combo?